### PR TITLE
Full screen toggle (fixes #2420)

### LINKED ISSUE #2420: Full screen on editor
There is full screen mode on markdown editor which if confusing to use.
screen shot 2018-10-30 at 07 59 13
Button on image leads to look like this on next image. And its not clear how to return back. You have to press **Esc** key get back
screen shot 2018-10-30 at 07 59 32

So we might want to clean some of these action

### PR DESCRIPTION
**Pull request by svlesiv**
fixes #2420 

## Screenshots
screen shot 2018-11-06 at 9 43 32 pm

screen shot 2018-11-06 at 9 43 20 pm

screen shot 2018-11-06 at 9 43 45 pm

### CODE REVIEW

**Comment by yuviii** [comment]
Left margin should be reduced i guess
@paulbert @lmmrssa 

screenshot 2018-11-07 13 21 33

File: src/app/shared/forms/planet-markdown-textbox.scss [hunk 1]
```diff
@@ -26,8 +26,8 @@ planet-markdown-textbox {
   }
 
   .CodeMirror.CodeMirror-fullscreen {
-    margin-top: 10px;
-    margin-left: 70px;
+    padding-top: 30px;
+    padding-left: 80px;
```
Comment (yuviii) [inline src/app/shared/forms/planet-markdown-textbox.scss]: padding left seems to be big. 10 px looks ok.

**Comment by lmmrssa** [comment]
@svlesiv nice work, looks good. There are also other action next to it. 
`Toggle Preview` `Toggle Side by Side` could you also test with these.

**Comment by svlesiv** [comment]
@lmmrssa thanks!) 
`Toggle Preview` and `Toggle Side by Side` doesn't work well. I will try to fix it.

File: src/app/shared/forms/planet-markdown-textbox.scss (hunk 1 above)
Comment (svlesiv) [reply src/app/shared/forms/planet-markdown-textbox.scss]: You are right.  Probably I would need to add some media queries for that.

**Comment by svlesiv** [comment]
screen shot 2018-11-07 at 5 15 17 pm

screen shot 2018-11-07 at 5 15 35 pm

**Comment by svlesiv** [comment]
screen shot 2018-11-07 at 5 53 19 pm

screen shot 2018-11-07 at 5 53 55 pm

**Comment by paulbert** [comment]
You can reference the way the component creators implemented it here:
https://teradata.github.io/covalent/#/components/text-editor

**Comment by svlesiv** [comment]
Hi @paulbert. Thanks for the link. I will take a look and try to implement that technic.

**Comment by paulbert** [comment]
I looked at this a little more and found what does it: check out the `text-editor-demo` element in their implementation.  It uses `position: absolute` to stretch the view to fill the screen and `transform: translateY(0px)` to set that as the containing element for the fixed position editor.  Works with the two navigation layouts as is.

**Comment by svlesiv** [comment]
Thanks for your advice @paulbert. I looked on a demo. Seems like planet code doesn't generate classes for animation. So I overwrote the values of existing classes.

**Pull request merged**